core/src/main/java/org/apache/carbondata/core/cache/CarbonLRUCache.java

```diff
@@ -217,6 +217,7 @@ public boolean put(String columnIdentifier, Cacheable cacheInfo, long requiredSi
     } else {
       synchronized (lruCacheMap) {
         addEntryToLRUCacheMap(columnIdentifier, cacheInfo);
+        currentSize = currentSize + requiredSize;
```

Comment:
it seems there is no validation for `requiredSize`, what if it is negative value?

Reply:
i dont think the required size can be negative.